modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_columns.yml

```diff
@@ -3,3 +3,6 @@ status: true
 dependencies: {  }
 id: grid_columns
 label: 'Grid columns'
+icon_uuid: null
+description: null
```

Comment:
can we have a meaningful description for a paragraph?

Reply:
Fixed